mix.exs

```diff
@@ -69,11 +71,17 @@ defmodule Hoplon.MixProject do
       "mix.exs",
       "README*",
       # "readme*",
-      "LICENSE*"
+      "LICENSE*",
       # "license*",
       # "CHANGELOG*",
       # "changelog*",
-      # "src"
+      "src"
+    ]
```

Comment:
maybe just uncomment all and keep the defaults, but explicit? They seem to be good.

Reply:
no, that doesn't work, it complains about the files being missing, let's leave it how it is.